modules/cert-manager-istio-csr-updating.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-integrating-istio.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="cert-manager-istio-csr-updating_{context}"]
+= Updating the {cert-manager-operator} with istio-csr 
+
+[IMPORTANT]
+====
+When the Istio-CSR TechPreview feature gate is enabled, the operator cannot be upgraded. To upgrade to the next available version, you must uninstall the Operator and remove all {cert-manager-operator} and Istio-CSR resources before reinstalling it.
+====
```

Comment:
I don't think this module should exist without any plain text. I get that this info is important, but without any plain text, there's not really a need to use an admonition (since there's nothing else to distinguish this info from). I would just make this plain text:
```suggestion
When the Istio-CSR TechPreview feature gate is enabled, the operator cannot be upgraded. To upgrade to the next available version, you must uninstall the Operator and remove all {cert-manager-operator} and Istio-CSR resources before reinstalling it.
```